camel/configs/sglang_config.py

```diff
@@ -0,0 +1,120 @@
+            (default: :obj:`True`)
+        regex (str, optional): A regular expression to constrain the output
+            tokens. (default: :obj:`None`)
+        json_schema (str, optional): JSON schema to constrain the output
+            structure. (default: :obj:`None`)
+        min_new_tokens (int, optional): Minimum number of new tokens to
+            generate. (default: :obj:`0`)
+        top_k (int, optional): The number of highest probability tokens to
+            consider during sampling. Setting it to a positive integer limits
+            the number of tokens to sample from. A value of `-1` means no
+            limit. (default: :obj:`-1`)
+        min_p (float, optional): Minimum probability for nucleus sampling.
+            Any token with a probability lower than `min_p` will be ignored.
+            (default: :obj:`0.0`)
+        repetition_penalty (float, optional): A penalty for repeating tokens.
+            Values greater than `1.0` discourage repetition, and values less
+            than `1.0` encourage repetition. (default: :obj:`1.0`)
+        logprob_start_len (int or list, optional): The starting token position
+            for returning logprobs. If `None`, only the logprobs for output
+            tokens will be returned. (default: :obj:`None`)
+        top_logprobs_num (int or list, optional): The number of top logprobs to
+            return at each position in the output sequence. If `None`, no
+            logprobs are returned. (default: :obj:`None`)
+        return_text_in_logprobs (bool, optional): Whether to include the
+            generated text in the logprobs output. (default: :obj:`False`)
+        stream (bool, optional): Whether to stream the generated output in
+            chunks. If set to `True`, the response will be streamed as it is
+            generated. (default: :obj:`False`)
+        max_tokens (int, optional): The maximum number of tokens to generate
+            in the chat completion. The total length of input tokens and
+            generated tokens is limited by the model's context length.
+            (default: :obj:`None`)
+    """
+
+    stop: Union[str, Sequence[str], NotGiven] = NOT_GIVEN
+    temperature: float = 1.0
+    top_p: float = 1.0
+    n: int = 1
+    frequency_penalty: float = 0.0
+    presence_penalty: float = 0.0
+    stop_token_ids: Optional[Sequence[int]] = []
+    max_new_tokens: int = 128
+    ignore_eos: bool = False
+    skip_special_tokens: bool = True
+    spaces_between_special_tokens: bool = True
+    regex: Optional[str] = None
+    json_schema: Optional[str] = None
+    min_new_tokens: int = 0
+    top_k: int = -1
+    min_p: float = 0.0
+    repetition_penalty: float = 1.0
+    logprob_start_len: Optional[Union[Sequence[int], int]] = None
+    top_logprobs_num: Optional[Union[Sequence[int], int]] = None
+    return_text_in_logprobs: bool = False
+    stream: bool = False
+    max_tokens: Union[int, NotGiven] = NOT_GIVEN
```

Comment:
From my personal point of view, there is no need to set so many parameters. You only need to set some commonly used parameters. For details, please refer to ollama

Reply:
@Wendong-Fan

Reply:
Thanks @raywhoelse , I believe it's fine to include as many parameters as possible; users are free to choose and use any of them as needed

Reply:
Also, SGLang's default parameter is generally good enough. Do not need to set it too complicated.